alacritty/src/wayland_theme.rs

```diff
@@ -32,51 +34,61 @@ impl AlacrittyWaylandTheme {
         }
     }
 
-    fn color_icon_color(&self, color: Rgb, status: ButtonState) -> [u8; 4] {
-        match status {
-            ButtonState::Hovered => [0xff, color.r, color.g, color.b],
-            ButtonState::Idle => [0xff, self.foreground.r, self.foreground.g, self.foreground.b],
-            ButtonState::Disabled => {
-                [0xff, self.dim_foreground.r, self.dim_foreground.g, self.dim_foreground.b]
-            },
-        }
-    }
-}
-
-impl WaylandTheme for AlacrittyWaylandTheme {
-    fn primary_color(&self, _window_active: bool) -> [u8; 4] {
-        [0xff, self.background.r, self.background.g, self.background.b]
-    }
-
-    fn secondary_color(&self, window_active: bool) -> [u8; 4] {
+    fn color_icon_color(&self, color: Rgb, status: ButtonState, window_active: bool) -> Rgb {
```

Comment:
What's the reason for this method name again? Looking at it it seems pretty silly.

Reply:
idk, it was named like that before.

Reply:
I'd probably change it, but I wouldn't require it for this PR.